config/routes.rb

```diff
@@ -1,6 +1,6 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-
-  # Defines the root path route ("/")
-  # root "articles#index"
+  get "users" => "users#index"
+  get "users/:id" => "users#show"
+  get "users/:id/posts" => "posts#index"
+  get "users/:id/posts/:id" => "posts#show"
```

Comment:
- [ ] [optional] It would be nice if you used resources for your routes, it makes your code shorter and easier to scale.
```suggestion
   resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end

```

Reply:
Thanks!